src/code42cli/util.py

```diff
@@ -167,3 +169,9 @@ def _get_default_header(header_items):
             if key not in header and isinstance(key, str):
                 header[key] = key
     return header
+
+
+def hash_event(event):
```

Comment:
`hash_json()` might be a more accurate name

Reply:
Since this is aimed at deterministically hashing events that don't have a unique ID in them, this should take a dict of the event data so it ensures the json is sorted before hashing. 

Maybe I should just call it `hash_dict` and raise an exception if it's passed a string?